planning/obstacle_stop_planner/include/obstacle_stop_planner/node.hpp

```diff
@@ -117,6 +117,8 @@ class ObstacleStopPlannerNode : public rclcpp::Node
 
   rclcpp::Publisher<VelocityLimit>::SharedPtr pub_velocity_limit_;
 
+  rclcpp::Publisher<sensor_msgs::msg::PointCloud2>::SharedPtr pub_obst_;
```

Comment:
[imo] I think it is hard to understand what `obst` mean. `pub_filtered_points_` is better...?

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/2624/commits/65a9d236b5c8c8b6fd4cab7cc697ad072c1ed7d0